src/rules/trailingCommaRule.ts

```diff
@@ -230,4 +230,16 @@ class TrailingCommaWalker extends Lint.AbstractWalker<Options> {
             this.addFailureAt(list.end - 1, 1, Rule.FAILURE_STRING_NEVER, Lint.Replacement.deleteText(list.end - 1, 1));
         }
     }
+
+    private isWrappedSingleExpression(list: ts.NodeArray<ts.Node>) {
+        if (list.length !== 1) {
+            return false;
+        }
+
+        const parentNodeChildren = list[0].parent!.getChildren();
+        const openTokenIndex = parentNodeChildren.findIndex((node) => node.pos === list.pos) - 1;
+        const openToken = parentNodeChildren[openTokenIndex];
+
+        return isSameLine(this.sourceFile, openToken.end, list[0].getStart());
+    }
```

Comment:
Besides the comments above, here's how I would have written this method:
```ts
return list.length === 1 &&
    // check if the expression ends on a line following the opening token
    !isSameLine(this.sourceFile, list.pos, list[0].end) && // list[0].end is important here, because list.end could include the trailing comma
    // check if expressions starts in the same line as the opening token - you already had this
    isSameLine(this.sourceFile, list.pos, list[0].getStart(this.sourceFile);
```